### ISSUE TITLE
REQUEST: New membership for arhell

### ISSUE THREAD

**Issue by Arhell** [tektoncd/community#773]
### GitHub Username

@arhell

#### issue:
https://github.com/tektoncd/website/issues/created_by/Arhell
https://github.com/tektoncd/website/issues?q=is%3Aissue+author%3AArhell+is%3Aclosed

https://github.com/tektoncd/results/issues?q=is%3Aissue+author%3AArhell
https://github.com/tektoncd/results/pulls/Arhell

#### PRs:

https://github.com/tektoncd/website/pulls/Arhell
https://github.com/tektoncd/website/pulls?q=is%3Apr+author%3AArhell+is%3Aclosed

https://github.com/tektoncd/results/pulls/Arhell
https://github.com/tektoncd/results/pulls?q=is%3Apr+author%3AArhell+is%3Aclosed

Basically I want to work on the site, fix bugs, improve user interface accessibility and more (for example, fixing links at least).

Unfortunately there are no review examples. Because there is not much activity on the site.

**Comment by dibyom** [comment]
+1 happy to endorse. Please open a PR adding yourself to the org.yaml file https://github.com/tektoncd/community/blob/main/org/org.yaml
